routes/api/events.js

```diff
@@ -59,6 +61,20 @@ router.get('/inactive', async (req,res)=>{
 
 });
 
+//endpoint para obtener eventos por id
+router.get('/:eventId',isAdmin, async (req,res)=>{
+    if (isNaN(req.params.eventId)) {
+        return responses.errorDTOResponse(res, 400, "El ID debe ser un número");
+    }
+    const event= await Event.findOne({
+        where:{id: req.params.eventId}
+    });
+     if(!event){
+        return responses.errorDTOResponse(res, 404, "No existe evento con este id");
+    }
+    return responses.singleDTOResponse(res,200,"Recuperación exitosa",event);
```

Comment:
```suggestion
    return responses.singleDTOResponse(res,200,"Evento recuperado con exito",event);
```
Mejor, más contextual